File: core/src/test/java/com.memoizlabs.kotlin_playground/currency/RxSchedulersForTest.java
package com.memoizlabs.kotlin_playground.currency;

import rx.Scheduler;
import rx.schedulers.Schedulers;

public class RxSchedulersForTest implements RxSchedulers {

    @Override
    public Scheduler getUiScheduler() {
        return Schedulers.immediate();
    }

    @Override
    public Scheduler getIoScheduler() {
        return Schedulers.immediate();
    }
}

File: core/src/test/java/com.memoizlabs.kotlin_playground/currency/RxSchedulersForTest.kt
package com.memoizlabs.kotlin_playground.currency

import rx.Scheduler
import rx.schedulers.Schedulers

class RxSchedulersForTest : RxSchedulers {
    override val uiScheduler: Scheduler = Schedulers.immediate()
    override val ioScheduler: Scheduler = Schedulers.immediate()
}
